src/components/common/MoreFilter.tsx

```diff
@@ -0,0 +1,92 @@
+import { MouseEvent, useState } from 'react';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+
+interface StyledMFProps {
+  option: string;
+  isActive: string;
+}
+function MoreFilter() {
+  const [isActive, setIsActive] = useState('문구·팬시');
+  const [isOpen, setIsOpen] = useState(false);
+  const toggle = () => setIsOpen((prevIsOpen) => !prevIsOpen);
+
+  const MoreFilterList = [
+    '문구·팬시',
+    '인테리어소품',
+    '주방용품',
+    '패션소품',
+    '공예품',
+    '인형·장난감',
+  ];
+
+  const handleClick = (e: MouseEvent) => {
+    if (!(e.target instanceof HTMLElement)) return;
+    e.stopPropagation();
+    setIsActive(e.target.innerText);
+    toggle();
+  };
```

Comment:
```suggestion
  const handleClick = (e: MouseEvent, option: string) => {
    if (!(e.target instanceof HTMLElement)) return;
    e.stopPropagation();
    setIsActive(option);
    toggle();
  };
```

이러한 방식으로도 할 수 있을 것 같아요!
이렇게 수정해줬으면 좋겠다는건 아닌데 저라면 innerText를 사용하기보다는 이렇게 했을 것 같다! 정도입니다 😄